utils/_context/_scenarios.py

```diff
@@ -946,12 +946,16 @@ class scenarios:
 
     default = EndToEndScenario(
         "DEFAULT",
+        weblog_env={"DD_DBM_PROPAGATION_MODE": "disabled"},
         include_postgres_db=True,
-        doc="Default scenario, spwan tracer and agent, and run most of exisiting tests",
+        include_mysql_db=True,
```

Comment:
Adding DB in a scenario slows it a lot, and we need to keep the default scenario as fast as possible. Let me one day to think about the best solution here.

Reply:
Thank you for the details below @cbeauchesne! Looking at how Postgres is the main DB that is being tested and service mode is the most used for .NET I have no objections: 
![image](https://github.com/DataDog/system-tests/assets/28125428/bfa47e38-9379-4e07-ad7e-0dbdd51aa4a7)

Will work on implementing these changes unless you have any other comments.

Reply:
Ok for me !